actions/kpops-runner/README.md

```diff
@@ -16,6 +16,7 @@ This action runs KPOps with the given config.
 | kpops-version     |    ❌    |    latest     | string | KPOps version to install                                                                                                                      |
 | helm-version      |    ❌    |    latest     | string | Helm version to install                                                                                                                       |
 | token             |    ❌    |    latest     | string | secrets.GITHUB_TOKEN, needed for setup-helm action if helm-version is set to latest                                                           |
+| python-version             |    ❌    |    "3.x"     | string | Python version to install (Defaults to the latest stable version of Python for specified major version)
```

Comment:
with `3.x` this might break when Python 3.12 is released and kpops is not updated yet

Reply:
How about 3.11.x?

Reply:
do you know what's the difference to 3.11?